README.md

```diff
@@ -13,7 +13,8 @@ Acknowledgements: Bhai Tarsem Singh UK, SHARE UK, Khalis Foundation, Khalsa Foun
 
 Powered by [<img height="30" src="http://www.banidb.com/wp-content/uploads/2018/03/full-banidb-logo.png">](http://banidb.com)
 
-[![Build Status](https://api.travis-ci.org/KhalisFoundation/sttm-desktop.svg?branch=release)](https://travis-ci.org/KhalisFoundation/sttm-desktop) [![Build Status](https://ci.appveyor.com/api/projects/status/github/khalisfoundation/sttm-desktop?branch=release&svg=true)](https://ci.appveyor.com/project/navdeepsinghkhalsa/sttm-desktop)
+[![Build Status](https://travis-ci.com/KhalisFoundation/sttm-desktop.svg?branch=release)](https://travis-ci.com/KhalisFoundation/sttm-desktop) 
```

Comment:
@tsingh777 veerji, from above links, the first one (https://api.travis-ci.com/KhalisFoundation/sttm-desktop.svg?branch=release) shows `build: unknown` to me.

Reply:
Yeah, there hasn't been a release branch build on travis-ci.com yet (there was a migration from travis-ci.org). It should auto update once we release a new version.